test/integration/clientset_test.go

```diff
@@ -219,6 +218,14 @@ func testBrokerClient(client servicecatalogclient.Interface, name string) error
 		)
 	}
 
+	// TODO: Here be dragons. Tests fail beyond this point due to known issues
+	// with our TPR-based storage implementation. Bail early (until those issues)
+	// are fixed, because some tests are better than no tests. If storage isn't
+	// TPR-based, carry on.
```

Comment:
for those watching, this work is captured in https://github.com/kubernetes-incubator/service-catalog/issues/599. @krancour is in-progress on that issue